lib/tctl/resources.go

```diff
@@ -68,6 +68,13 @@ func (res *streamResource) UnmarshalJSON(raw []byte) error {
 
 	var resource types.Resource
 	switch header.Kind {
+	case types.KindNode:
```

Comment:
Does this relate to the tests?

Reply:
Yes
We are querying all the nodes to ensure we can proceed with the test here
https://github.com/gravitational/teleport-plugins/blob/7866f13d8cbf60db86f5390db41563c90853db70/lib/testing/integration/setup.go#L122